tests/test_dem_fix.py

```diff
@@ -67,10 +67,10 @@ def test_schism_grid(tmpdir, natural_earth, dem_source, window):
 @DEM_SOURCES
 @WINDOWS
 def test_d3d_grid(tmpdir, natural_earth, dem_source, window):
-    grid = pg.grid(type="r2d", geometry=window, resolution=0.1, rpath=str(tmpdir) + "/")
+    grid = pmesh.set(type="r2d", geometry=window, resolution=0.1, rpath=str(tmpdir) + "/")
```

Comment:
we probably want to rename this variable too

Reply:
This one is tricky because the d3d mesh is actually a grid (r2d). However in the future this will change so we might as well be consistent.